packages/grafana-ui/src/components/FormattedValueDisplay/FormattedValueDisplay.story.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { FormattedValueDisplay } from './FormattedValueDisplay';
+import { withCenteredStory } from '@grafana/ui/src/utils/storybook/withCenteredStory';
+import mdx from './FormattedValueDisplay.mdx';
+
+export default {
+  title: 'Visualizations/FormattedValueDisplay',
+  component: FormattedValueDisplay,
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const basic = () => {
+  return <FormattedValueDisplay value={{ text: 'Test value' }} style={{ fontSize: 12 }} />;
+};
```

Comment:
This could use more examples or knobs?

Reply:
That' a good idea, just couldn't figure out all the use cases myself :)